winter-website.css

```diff
@@ -1,4 +1,70 @@
 body {
+	padding: 0px;
+	margin: 0px;
 	color: #e8f3fc;
 	background-color: black;
+	font-family: sans-serif;
+}
+
+*, *::before, *::after {
+   box-sizing: border-box;
+}
+
+.navBar {
+	display: flex;
+	justify-content: space-between;
+	width: auto;
+	padding: 20px;
+	margin: 20px;
```

Comment:
using px for sizing can cause issues down the line as it isnt a relative size. 
best practice is to set the body fontsize to 10px and then use rem for all other sizings. 
in this instance it would be 2rem not 20px

Reply:
yes! : ] fixed the font sizes from px to rem